pkg/shell/superuser.jsx

```diff
@@ -144,27 +144,25 @@
 
         title = _("Switch to administrative access");
         body = (
-            <>
-                { error && <><Alert variant={errorVariant || 'danger'} isInline title={error} /><br /></> }
-                <Form isHorizontal onSubmit={event => { apply(); event.preventDefault(); return false }}>
-                    { prompt.message && <span>{prompt.message}</span> }
-                    <FormGroup
-                        fieldId="switch-to-admin-access-password"
-                        label={prompt.prompt}
+            <Form isHorizontal onSubmit={event => { apply(); event.preventDefault(); return false }}>
+                { error && <Alert variant={errorVariant || 'danger'} isInline title={error} /> }
+                { prompt.message && <span>{prompt.message}</span> }
+                <FormGroup
+                    fieldId="switch-to-admin-access-password"
+                    label={prompt.prompt}
+                    validated={!error ? "default" : validated || "error"}
+                >
+                    <TextInput
+                        autoFocus // eslint-disable-line jsx-a11y/no-autofocus
+                        id="switch-to-admin-access-password"
+                        isDisabled={busy}
+                        onChange={setValue}
+                        type={!prompt.echo ? 'password' : 'text'}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221203-085317-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)